packages/app/src/pages/gemeente/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { colors, GmCollectionHospitalNiceChoropleth, gmData } from '@corona-dashboard/common';
+import { colors, ArchivedGmCollectionHospitalNice, gmData } from '@corona-dashboard/common';
```

Comment:
```suggestion
import { colors, ArchivedGmCollectionHospitalNice, gmData } from '@corona-dashboard/common';
``` 
would need to be changed to  ArchivedGmCollectionHospitalNiceChoropleth which is what I see in ```data.ts```

Reply:
Good catch!